strings/charactes_unique.py

```diff
@@ -18,23 +27,31 @@ def isUniqueSet(s):
         hashSet.add(i)
     return True
 
-def isUniqueHash(s):
+
+def isUniqueHash(s: str) -> str:
```

Comment:
Avoid using a single character as a variable names for variables whose scope extends just a few lines. Try making it more descriptive. `string`

Reply:
Strings is the name of the directory,string is the name of a module in the Python standard library, and str is the name of a Python builtin data type so in this case, s is an acceptable.